mkdocs.yml

```diff
@@ -71,11 +71,11 @@ pages:
   # API Permissions: api/permissions.md         # page-tree = NEED_NEW_PAGE
   - API Chaining: api/chaining.md
   # API Changes: api/changes.md                 # page-tree = NEED_NEW_PAGE
+- Setup:
```

Comment:
Sure, that makes more sense to me, I put it at top level because I thought there was a restriction to the number of subfolders based on what it says [here](https://docs.civicrm.org/dev/en/master/best-practices/documentation-style-guide/#parts-chapters-sections)

I'll move it into the hooks section though if you're happy with having it there.